references/classification/train.py

```diff
@@ -182,7 +188,12 @@ def load_data(traindir, valdir, args):
 def main(args):
     if args.output_dir:
         utils.mkdir(args.output_dir)
-
+        
+    if args.seed is None:
+        # randomly choose a seed
+        args.seed = random.randint(0, 2 ** 32)
+    utils.set_seed(args.seed)
+    
```

Comment:
What is the reason we need to set the seed? We don't do this anywhere else. Shall we remove it?

Reply:
I used it to perform a fixed ablation earlier on when experimenting with various configurations. I assume it's useful when there's no clear way of reproducing a specific experimentation and we want to remove seed randomness from the equation when quantifying effects.

Reply:
I see, so some features are still from the moment you did the ablations. Now that this is completed, we should remove them. This simplification is a standard practice for every model that we port.